src/common/hooks/useBatchGroup/useBatchGroup.js

```diff
@@ -0,0 +1,20 @@
+import { useQuery } from 'react-query';
+
+import { useOkapiKy } from '@folio/stripes/core';
+
+import { BATCH_GROUPS_API } from '../../constants';
+
+export const useBatchGroup = (batchGroupId) => {
+  const ky = useOkapiKy();
+
+  const { isLoading: isBatchGroupLoading, data: batchGroup = {} } = useQuery(
+    [BATCH_GROUPS_API, batchGroupId],
+    () => ky.get(`${BATCH_GROUPS_API}/${batchGroupId}`).json(),
+    { enabled: Boolean(batchGroupId) },
+  );
+
+  return ({
+    isBatchGroupLoading,
```

Comment:
maybe just `isLoading`?

Reply:
but in this case we still require to give this a proper name in composed useData hook. To me this looks more cleaner